src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -308,6 +310,44 @@ class RunSetupPyResult:
     output: Digest  # The state of the chroot after running setup.py.
 
 
+class FirstPartyDependencyVersionScheme(Enum):
```

Comment:
should this be using `@unique` ?

Reply:
Sure. We don't normally use it, but seems consistent with defensive programming.